2021-06-18 03:56:18,642 INFO    MainThread:4017874 [internal.py:wandb_internal():91] W&B internal server running at pid: 4017874, started at: 2021-06-18 03:56:18.641975
2021-06-18 03:56:18,652 DEBUG   HandlerThread:4017874 [handler.py:handle_request():124] handle_request: run_start
2021-06-18 03:56:18,654 INFO    WriterThread:4017874 [datastore.py:open_for_write():80] open: /home/chengzilong/DDQN/wandb/offline-run-20210618_035616-2blgqwdp/run-2blgqwdp.wandb
2021-06-18 03:56:20,914 DEBUG   HandlerThread:4017874 [meta.py:__init__():39] meta init
2021-06-18 03:56:20,914 DEBUG   HandlerThread:4017874 [meta.py:__init__():53] meta init done
2021-06-18 03:56:20,915 DEBUG   HandlerThread:4017874 [meta.py:probe():210] probe
2021-06-18 03:56:20,921 DEBUG   HandlerThread:4017874 [git.py:repo():33] git repository is invalid
2021-06-18 03:56:20,921 DEBUG   HandlerThread:4017874 [meta.py:_save_pip():57] save pip
2021-06-18 03:56:20,923 DEBUG   HandlerThread:4017874 [meta.py:_save_pip():71] save pip done
2021-06-18 03:56:20,924 DEBUG   HandlerThread:4017874 [meta.py:_save_conda():78] save conda
2021-06-18 03:56:26,048 DEBUG   HandlerThread:4017874 [meta.py:_save_conda():86] save conda done
2021-06-18 03:56:26,048 DEBUG   HandlerThread:4017874 [meta.py:probe():252] probe done
